cmd/topicctl/subcmd/apply.go

```diff
@@ -99,6 +100,12 @@ func init() {
 		false,
 		"Skip confirmation prompts during apply process",
 	)
+	applyCmd.Flags().BoolVar(
+		&applyConfig.ignoreFewerPartitionsError,
+		"ignore-fewer-partitions-error",
+		false,
+		"Avoid returning error when topic's config specifies fewer partitions than the current topic has",
```

Comment:
Might I suggest a more direct wording:

"Don't return error when topic's config specifies fewer partitions than it currently has"

Reply:
Sure thing. Will reword.

Reply:
@hhahn-tw Done!
Can you please approve?